airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -112,6 +112,7 @@ def __init__(
 
         kwargs["client_type"] = "glue"
         super().__init__(*args, **kwargs)
+        self.logs_hook = AwsLogsHook(aws_conn_id=self.aws_conn_id)
```

Comment:
Better move it out of the Hook constructor, into the property because this is optional and not requiredo all operations.

And in addition it also a good idea to propagate also all generic parameters, like region_name, verify and config

Reply:
Thanks for your feeback :)
By property do you mean class property with @property decorator ?

Reply:
Yeah or `cached_property` from `functools` which usual uses for hooks